content/en/blog/_posts/2022-12-13-host-process-containers-ga/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Windows HostProcess Containers Are Going Generally Available"
+date: 2022-12-13
+slug: windows-host-process-containers-ga
+---
+
+**Authors**: Brandon Smith (Microsoft) and Mark Rossetti (Microsoft)
+
+The long-awaited day has arrived: HostProcess containers, the Windows equivalent to Linux privileged
+containers, has finally made it to **GA in Kubernetes 1.26**!
+
+What are HostProcess containers and why are they useful?
+
+Cluster operators are often faced with the need to configure their nodes upon provisioning. Whether it's
+installing Windows services, configuring registry keys, managing TLS certificates,
+making network configuration changes, or even deploying monitoring tools such as a Prometheus's node-exporter.
```

Comment:
Prometheus's windows_exporter
Also may want to add a link: https://github.com/prometheus-community/windows_exporter/blob/master/kubernetes/kubernetes.md
node exporter is for linux

Reply:
good catch. Could you open PR for these fixes? They would got live as soon as we merge it, if I understand correctly